src/EditorFeatures/TestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -79,9 +71,29 @@ public Test()
             /// </summary>
             public OptionsCollection Options { get; } = new OptionsCollection(LanguageNames.CSharp);
 
+            protected override string DefaultFilePathPrefix
+            {
+                get
+                {
+#if CODE_STYLE
```

Comment:
i really don't like how infectious this seems to be.  it feels like we're becoming a c++ codebase with ifdefs everywhere to support conditional compilation.